crates/client/src/client/schema/primitives.rs

```diff
@@ -177,7 +177,7 @@ impl LowerHex for TxPointer {
     }
 }
 
-#[derive(cynic::Scalar, Debug, Clone)]
+#[derive(cynic::Scalar, Debug, Clone, PartialEq, Eq)]
```

Comment:
Right, it doesn't seem like these are needed anymore. I probably added them while iterating on the code, but I can cut them out. Though since this is a public type, I'd argue that we still should proactively add these common traits as per https://rust-lang.github.io/api-guidelines/interoperability.html#types-eagerly-implement-common-traits-c-common-traits

I can still cut them out if you want me to. Let me know what you think.

Reply:
I don't have a strong opinion here, leaving this up to you :-)

Reply:
Alright, then I'll keep them around. It's unclear if this PR will me merged as for now, with the current benchmarks.